FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -1782,16 +1795,16 @@ subroutine GFS_physics_driver                                 &
 
 !  --- ...  surface energy balance over ocean
 
-          call sfc_ocean                                                &
+           call sfc_ocean                                                &
```

Comment:
Looks like some formatting/whitespace diff made it into the PR. Can you revert to keep the diff minimal?

Reply:
Ah good catch! Thanks for pointing this out, I'll scan and fix!